src/AdminDashboard/SearchEngine/Panel/Panel.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import PanelResult from './PanelResults';
+import PanelItemType from './PanelItemType';
+import PanelFields from './PanelFields';
+
+
+class Panel extends Component {
+  render() { 
+    const fields = [];
+
+    this.props.itemResults.length && fields.push.apply(
+      fields, this.props.itemResults[0].map(field => {
+        return field['fieldName'];
+      })
+    )
+    
+    return (
+      <React.Fragment>
```

Comment:
Did you try using the short syntax?

https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Could the version of create-reaction-app not be compatible with this syntax?

![image](https://user-images.githubusercontent.com/12489333/35783090-6786de1c-09d8-11e8-9b91-4e540e6f0ff7.png)

Reply:
It is possible, it is necessary to review